manifests/charts/base/templates/clusterrole.yaml

```diff
@@ -119,13 +119,16 @@ rules:
     resources: ["*"]
     verbs: ["get", "list", "watch"]
   - apiGroups: [""]
-    resources: ["endpoints", "pods", "services", "nodes", "replicationcontrollers"]
+    resources: ["endpoints", "pods", "services", "nodes", "replicationcontrollers", "namespaces"]
     verbs: ["get", "list", "watch"]
   - apiGroups: ["apps"]
     resources: ["replicasets"]
     verbs: ["get", "list", "watch"]
   - apiGroups: ["authentication.k8s.io"]
     resources: ["tokenreviews"]
     verbs: ["create"]
+  - apiGroups: [""]
+    resources: ["configmaps"]
+    verbs: ["create", "get", "list", "watch", "update"]
```

Comment:
nit: having a "istio-reader" role with write access is a bit misleading. Although it already has tokenreviews write access..

Reply:
+1 - using a new name would also be good for allowing both use cases, the 'central cluster in control' is very different than 'each cluster running istiod and just reading endpoints'. 

I understand this may require changes in docs/istioctl/etc to use the right SA for the case - but this change breaks the fundamental security assumptions of MC 'only reads endpoints'.

For tokenreview - I haven't reviewed that, but same security considerations should be used. 
If tokenreview 'write' is only used to validate - it might be equivalent with a read (with a horrible API). If it can be for anything else - then separate role and SA should be used.

Reply:
add resourcename to restrict the usage